examples/outcomes/1/outcomes-lb.json

```diff
@@ -1,9 +1,9 @@
 {
-  "project_id": 1780659,
+  "project_id": 1,
   "logging.googleapis.com/labels": {
     "host": "lb-6"
   },
-  "org_id": 0,
+  "org_id": 1,
```

Comment:
nit: I'd leave this one as 0. Three was no org Id in this case so it filled in 0